app/src/main/java/com/hover/stax/utils/AnalyticsUtil.kt

```diff
@@ -0,0 +1,107 @@
+package com.hover.stax.utils
+
+import android.content.Context
+import android.os.Bundle
+import com.amplitude.api.Amplitude
+import com.amplitude.api.Identify
+import com.appsflyer.AppsFlyerLib
+import com.google.firebase.analytics.FirebaseAnalytics
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.ktx.initialize
+import com.hover.sdk.api.Hover
+import com.hover.stax.BuildConfig
+import org.json.JSONException
+import org.json.JSONObject
+import timber.log.Timber
+import java.util.*
+
+class AnalyticsUtil {
```

Comment:
You can change this to an `object` instead of a class. That way the underlying functions are static by default. You wouldn't need to wrap them in a `companion object`.